### ISSUE TITLE
Bug: `0.2.6` of `@eslint/plugin-kit` missing from npm

### ISSUE THREAD

**Issue by voxpelli** [eslint/rewrite#153]
### Which packages are affected?

- [ ] `@eslint/compat`
- [ ] `@eslint/config-array`
- [ ] `@eslint/core`
- [ ] `@eslint/migrate-config`
- [ ] `@eslint/object-schema`
- [x] `@eslint/plugin-kit`

### Environment

npm web page: https://www.npmjs.com/package/@eslint/plugin-kit

### What did you do?

I found an unexpected duplicated dependency when reviewing the lockfile maintenance in neostandard: https://github.com/neostandard/neostandard/pull/256

### What did you expect to happen?

That a single version of `@eslint/core` was included

### What actually happened?

Since [`0.2.6`](https://github.com/eslint/rewrite/releases/tag/plugin-kit-v0.2.6) is missing from npm the plugin-kit correctly got its own older `@eslint/core`, separate from the main one

### Link to Minimal Reproducible Example

https://www.npmjs.com/package/@eslint/plugin-kit?activeTab=versions

### Participation

- [ ] I am willing to submit a pull request for this issue.

### Additional comments

Kind of hard to make a reproduction of a lack of publishing to npm, I hope you can tolerate this report anyhow

**Comment by nzakas** [comment]
Yep, there was an error during the last release and `plugin-kit` wasn't published. There's not an easy way to fix that without publishing manually, which we don't really like doing. We'll take a look.

**Comment by fasttime** [comment]
Note that the same issue was reported here: https://github.com/eslint/rewrite/pull/147#issuecomment-2644538157

**Comment by voxpelli** [comment]
@fasttime If you would structure the publishing of each package and destination as separate jobs rather than as steps within the same job – then you could rerun just the failed jobs to give it another go. See eg: https://github.com/voxpelli/eslint-config/actions/runs/12120540417

If you don't want to do that, then an alternative would be to add `continue-on-error: true` to the publish steps – then at least other steps will not become collateral damage if one fails

**Comment by fasttime** [comment]
> [@fasttime](https://github.com/fasttime) If you would structure the publishing of each package and destination as separate jobs rather than as steps within the same job – then you could rerun just the failed jobs to give it another go. See eg: https://github.com/voxpelli/eslint-config/actions/runs/12120540417
> 
> If you don't want to do that, then an alternative would be to add `continue-on-error: true` to the publish steps – then at least other steps will not become collateral damage if one fails

Thanks for the suggestions @voxpelli. The current release job contains a single `release-please-action` step that's responsible for determining which packages should be released. It does so by setting environment variables that only exist in the current job run, so I'm not sure if splitting the release in multiple re-runnable jobs is possible. If it is, that would be perfect.

Adding `continue-on-error: true` to the publish steps sounds like a good idea. @nzakas what do you think?

**Comment by voxpelli** [comment]
@fasttime [Same here](https://github.com/voxpelli/ghatemplates/blob/505e23d97d17e657b212d6e3635c6ad98f0a1e95/.github/workflows/release-please-4.yml#L36-L49), you can add this at the job level to separate it between jobs:

```yaml
jobs:
  release_please:
    outputs:
      'packages/compat--release_created': ${{ steps.release.outputs['packages/compat--release_created'] }}
      'packages/config-array--release_created': ${{ steps.release.outputs['packages/config-array--release_created'] }}
      'packages/core--release_created': ${{ steps.release.outputs['packages/core--release_created'] }}
      'packages/migrate-config--release_created': ${{ steps.release.outputs['packages/migrate-config--release_created'] }}
      'packages/object-schema--release_created': ${{ steps.release.outputs['packages/object-schema--release_created'] }}
      'packages/plugin-kit--release_created': ${{ steps.release.outputs['packages/plugin-kit--release_created'] }}
```

Then [on individual jobs](https://github.com/voxpelli/ghatemplates/blob/505e23d97d17e657b212d6e3635c6ad98f0a1e95/.github/workflows/release-please-4.yml#L55-L56) do:

```yaml
  compat:
    if: needs.release_please.outputs['packages/compat--release_created']
    needs: release_please
```

Added bonus: The releases for all packages will then happen in parallel

**Comment by voxpelli** [comment]
You could also probably wrangle it into a matrix – that way you wouldn't have to duplicate the publish jobs for every package

**Comment by nzakas** [comment]
I'm working on a new publish script that will solve a lot of the issues we have:
https://github.com/eslint/rewrite/pull/151

**Comment by voxpelli** [comment]
@nzakas Will it have an improved failure recovery?

**Comment by nzakas** [comment]
Check out the PR and see for yourself :smile:

**Comment by mahnunchik** [comment]
Any news?

**Comment by nzakas** [comment]
@mahnunchik all updates are posted on the issue. There's no need to ask.

**Comment by nzakas** [comment]
The package was published:
https://www.npmjs.com/package/@eslint/plugin-kit
